config/analytics.yml

```diff
@@ -1,8 +1,9 @@
 ---
 shared:
-  feedbacks:
+  user_feedbacks:
     - id
+    - topic
     - rating
-    - successful_visit
+    - explanation
```

Comment:
Any data model changes need to be communicated to the data insights team, by send a summary of the changes to the slack channel: #twd_data_insights

Reply:
I have done that 👍